sdk/src/idl/drift.json

```diff
@@ -1921,6 +1921,11 @@
           "name": "oracle",
           "isMut": false,
           "isSigner": false
+        },
+        {
+          "name": "spotMarketVault",
+          "isMut": false,
+          "isSigner": false
```

Comment:
this keeps happening (changes from other prs) cause anchor test suite isnt ran locally in branch before merge?